client/src/components/ElementEditor/ElementEditor.js

```diff
@@ -105,21 +101,15 @@ class ElementEditor extends PureComponent {
           dragSpot={dragSpot}
           isDraggingOver={isDraggingOver}
           dragTargetElementId={dragTargetElementId}
+          sharedObject={sharedObject}
         />
         <ElementDragPreview elementTypes={elementTypes} />
-        <input
```

Comment:
This `<input>` was how the elemental data was POSTed along with the rest of the parent page data